Taxi/Taxi/Presenter/SignUpCodeView.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  SignUpCodeView.swift
+//  Taxi
+//
+//  Created by sanghyo on 2022/06/14.
+//
+
+import SwiftUI
+
+struct SignUpCodeView: View {
+    @Environment(\.dismiss) private var dismiss
+    @State private var signUpCode: String = ""
+    @State private var codeState: FieldState = .normal
+    @State private var isActive: Bool = false
+    private let developerCode: String = "popopot"
+    @FocusState private var focusField: Bool
+
+    var body: some View {
+        NavigationView {
+            VStack {
+                VStack(alignment: .leading, spacing: 80) {
+                    Text("가입코드를 입력해주세요")
+                        .signUpTitle()
+                    UnderlinedTextField(text: $signUpCode, codeState, "가입코드")
+                        .font(Font.custom("AppleSDGothicNeo-Regular", size: 20))
```

Comment:
signUp 화면에는 텍스트 익스텐션이 전부 있는 것으로 알고 있습니다만...!

```suggestion
                        .signUpPlaceholder()
```